### PR TITLE
fix euler number

### PR DESCRIPTION
**Pull request by latot**
Fixes https://github.com/cbm755/octsympy/issues/591

### CODE REVIEW

**Comment by latot** [comment]
test pass in python 3.4, 2.7 timeout.

File: inst/@sym/sym.m [hunk 1]
```diff
@@ -299,6 +299,12 @@
       x = 'lamda';
     elseif (strcmp(x, 'Lambda'))
       x = 'Lamda';
+    elseif (strcmp(x, '-e'))  %%Special case, don't assign e to E.
+      s = python_cmd ('return -E');
```
Comment (cbm755) [inline inst/@sym/sym.m]: why not `x = '-E'` like other cases

File: inst/@sym/sym.m [hunk 2]
```diff
@@ -299,6 +299,12 @@
...
+      return
+    elseif (strcmp(x, 'e'))
+      s = python_cmd ('return E');
+      return
     end
 
     % Decide whether to pass to S() or Symbol()
```
Comment (cbm755) [inline inst/@sym/sym.m]: possible need to add `E` below?

File: inst/@sym/sym.m [hunk 3]
```diff
@@ -693,3 +699,9 @@
 %! else
 %!   delete([myfile '.mat'])
 %! end
+
+%!test
+%! a = sym(e);
```
Comment (cbm755) [inline inst/@sym/sym.m]: This `sym(e)` test will fail on Matlab: needs wrapped in the `if exist OCTAVE_VER` construct.

**Comment by latot** [comment]
hi, i considere this pr as deprecated, please check this: https://github.com/cbm755/octsympy/pull/594
we can keep this open to check or decide some things

**Comment by cbm755** [comment]
@latot are you likely to redo this on top of master?

**Comment by cbm755** [comment]
Closing in favour of #824, which is similar to this but done on top of master.

**Pull request closed without merging**